src/scripts/validation.js

```diff
@@ -93,4 +85,9 @@ const resetFormAndButton = (formEl, config) => {
   disableButton(submitButton, config.inactiveButtonClass);
 };
```

Comment:
You have `resetValidation` and `resetFormAndButton`. Please, use one of them in `index.js`